services/config/ffc-sfd.yaml

```diff
@@ -2,5 +2,8 @@ topics:
   - name: ffc-sfd-messages
     subscriptions:
       - ffc-sfd-messages-processor
+  - name: ffc-sfd-customer-receiver
```

Comment:
I think we should split these concerns, rather than have a single topic for all update types.

we could have the following topics

- `ffc-sfd-customer`
- `ffc-sfd-application`
- `ffc-sfd-payment`

Then each can have it's own list of independent subscriptions.

Then they're free to evolve independently and all aren't stuck in the same queue.

For example, with messages, we know there are going to be two consumers, `ffc-sfd-message-receiver` and `ffc-sfd-message-processor`.

We don't want the message receiver service to have to deal with a load of customer updates too.

Reply:
Have updated naming of topic and subscription to reflect this, thanks.